src/ParallelDownloader.php

```diff
@@ -244,7 +244,7 @@ protected function getRemoteContents($originUrl, $fileUrl, $context, array &$res
             return $result;
         }
 
-        if (!$this->downloader || !preg_match('/^https?:/', $originUrl)) {
+        if (!$this->downloader || !preg_match('/^https?:/', $fileUrl)) {
```

Comment:
`$originUrl` is always `github.com` in my case.

and

`$fileUrl` is something like `https://codeload.github.com/symfony/service-contracts/legacy.zip/ffc7f5692092df31515df2a5ecf3b7302b3ddacf?access_token=60f62f9b058d01dd4c051bda82394bc5d50b5190`

Reply:
@kevinpapst this will also still solve the issue you fixed in https://github.com/symfony/flex/pull/561, right?

Reply:
Yes, definitely. All that was missing was the check for https.